Sources/Player/PlayerItem.swift

```diff
@@ -8,6 +8,18 @@ import AVFoundation
 import Combine
 
 /// An item to be inserted into a player.
+///
+/// This class represents a playable item that can be inserted into a ``Player``.
+/// It provides convenient initialization methods for different types of assets, including simple media, custom resource loading, and encrypted content.
+///
+/// Three provided items are as follows:
+///
+/// - Simple: ``simple(url:metadata:trackerAdapters:configuration:)``, which represents a playable item with a URL.
+/// - Custom: ``custom(url:delegate:metadata:trackerAdapters:configuration:)``, which represents an item with custom resource loading.
+/// - Encrypted: ``encrypted(url:delegate:metadata:trackerAdapters:configuration:)``, which represents an item loaded with a content key session, 
+/// suitable for playing encrypted content.
+///
+/// - Note: You can also create your own ``PlayerItem`` by extending the class.
```

Comment:
Could probably be a bit less redundant, something like:

```suggestion
///
/// This class represents a playable item that can be inserted into a ``Player``. 
///
/// It provides convenient initialization methods for different types of assets:
///
/// - Simple assets which can be played from a simple URL.
/// - Custom assets which require custom resource loading.
/// - Encrypted assets which require a FairPlay content key session.
///
/// - Note: You can also create your own ``PlayerItem`` by extending the class.
```

As for the `Asset` we could then organize methods in groups for each category.